.travis.yml

```diff
@@ -23,3 +23,26 @@ script:
+# Upload master wheels to S3 for pypicloud
+deploy:
+  provider: s3
+  access_key_id:
+#    secure: "qAZ9Bb1bQws64yeA2CHMzyl6qNo+EFGEkEi85L2beC9R2lADKHnrkoerpbDuaZoYhNR2lyxRnmGRDMxcqYMXvYrNEz1i96TjP3SiaXqVwnNtF/LVuwnjWM54dJA/Bd/jZvBolJAnLmZdfbFqwyOlIq7ewdZADnk0WjxEpWM+wzUnqOJi2us4ZxZKZ4SJkyEcmX1phG34pl8cOLshfWukE11ESYPOEQZBoRMQsrkyEhfiCsz8KjsG05ufR9dk/jyZrllPB/UPQnORciHHAspgEc6068n7jdgTTlntcS7/jsHC27v3R+UQynXgayN1A8KUw1BtaZQxbQgA+dP2bWqttQOQeZOfuDc/fxFk7FiqMjJPgNAvr16GrK1zR5cIo+ErWeJ2k7WFy66QIcJxHI2qywbGuFNDhxFU0LI+/6rgwHC20t2gmLjMWvz79q+cb7MyosVf5Zi5AknVUrayb1z83WV5OSE7LOyWgdTcWFAHTfqe+08H+La2YcNWKyCPz94R7W1zLbhqEA2m2fJCqQieNqzWOmVYaHZDTlZTGw1dOuadCKja5NBHivuTLoDjeOQQg6/B7oEHwLDmWff10ij52B+klgDo5XLT/BFwnkV/sH1V9AhB6fZERR0v2VMqhBWwd9BvY+8K9izLlZkSKHzjlUBru7ce3cmiRRkNwZH4erA="
+    secure: "BOUzAjnu00Q0iHcZG0V8irkWbvgGn006tSbWBChld8hI0dP0RHwkAyjiiEYQ+vBrajIjZ7jUXdc8/ZO7AR3EJ+oQvQtiMn3TIaxSNgNW/O17USPxJZR9b6j+hw9oLr0DjeyIGQlmnywOV3Tz2ZqItcO3BUb3PXpGavPstkmS15+966s69u0YBnpxyq+l0TsFgVFgMIsUAeOZwrweVWy+iMR/fOdekgRI9VCpbmyakYVO4aQVvjyqmVA9IvuFphrf7+mR2aFExoQpD9JsiKUJ5cddKrbYvVRYbjDfsN2fJafESsozZJzlbpzbth7WbqfoK1j4VpvZfqoH5JEKyKiF0CAbH7VrM6ogcJxg6hFyjxmLgk9HJW6BdpEmmxEitLOAYWqdbBTnYoB1oORgRor27PVXLWLHjeJMnP8UedQIqAquh+WL+CR26SQzP9wGSIR54VAOo92lHVKnfVF4NpBryiaQDkQvYcKxcpuspCfe2nnAE+e/uad4zKbX9rMQXwLHf2BSwSq3CZS+9snLPMNZbCiZRXxwffJkYCVNUBxx1qh6YmTSEodpsZkrXJGj6SWXog745hXtNJVmSVxmYIxuFG7SOxSlInActhEravqQJ79ujTdXUUo29W9sMJ6ZQ4RdIFfsua9IXR4sWkwII/MC1jBbSbCBGXTb9up9v7ZL1NM="
+  secret_access_key:
+#    secure: "kjtJraeeJvdcCSjsSykPp77lPxYcs9gINqyuaPFZaj6E4Tr+9UTSiGRwiNl9dwyS/TZR8qXchQozXpQxJVGMcrJpGB7eJiwM75paQ+3rUWYqTiaonvt88uhLn6I6XyBCCDcHeh5JQxCt8CmXP5u4bYgQKXRbh55soBYqRfQRAIl6Kq2P/n6j3Ge54flB1iGsq48LjsicOBfz+vXthtJRB06iGFV7HHIfkSLgE7UC1BiOiWsBq5lTL8NKq6/sSx39i8CIuDN5V7I4IY9EwjzjPK/D4olQ/sJ/3106Xu/q5R05YF9qgG243W+o5iOjio2o1bjPvM7tjlredPVPiN70d9Z1h5F+Pq+iv3yebRG7hDKEBjnOnERbuL6UtyrkCXqW7ySA4/XgP5XzniHOmZA9NopRM93ZD3/gkUVBGEKxjPIa/PP4HvRmqcunfhoAPWTwX2XztjWtu7aWiwUdVrybUG32MAzTzJbC1LvlYEUHu+VkEjUEYqzi8SWRDQv+UzwfhJfdYSwYoZYHbQUwc8EJP2vCQ9ac9+gDv+ibKsFrfAyd7sHG+mugpfb+qGC+SyaKPxgy8HG9OG4tVED78kGQrQP60/HJLa+Fie9SbzJER4wXpS+Qmm39QYL9RToqk01n1/s2yXvLaPngz7pAz30SqNycK/CM+KkGPnoS1ypgkGA="
+    secure: "Ipp50Lq+/x03czeUOpm7SP1Q+Zcn47LPrGgJsO2guwP1Y79ZvWw1ISgXQSRqkOO6I1OZVowS33J0qDda4UTn07EA4TnC73p1oG5RP7OTyz03yXXd20VJyXHvczfiXOrIDULylcRv2YQ4xojTh7opi6Lb3vhlLSgd4sPJJDqrkMeRiHf4K1s/8K8tbBFhPxXnXB+CCftgTC+f1nGZukfMcAmCzvnqCIqwZVrtXO3kwPTs40kKZWqMaDJSArbdOob21ydXuk+ajpC54Zkksl1hlJVKOASwLaDpIJSUOGpqB9iLwyqbtRy68PfZ7BMA/Ga0ZmLIOl2itVIZ0O7mrXem5Oz1t3DhApaa1D/YAuZISiK8sGm/m+nqim8XsbuISZclS4RMQliIwswuo1T1/kDdqxijBnHIq2vZhtba6qjaRYFgeg6pnoMYIjds9mpWOz0ilyAB5wQG9ZyJpjwELxEvMmsbo5PoeRWoXX2G3U9HeQWLb2fQY/o361eIWXrFUakYryOEdJvliTCRlUu7tR4i4FTAxhdLAPV51ngEJRfwlp4/cqT4fZHfNPYqp75JEb9pWfgG4e8r5sTrZxN+muucL+FWlVuF7r+SP6AE7ar8nFHQlwcxFLeUimVS3nj0Bjk/Rpt1aYOZ4mx/ihaUyDvGJqvrBRZ2vklT37YEeS/28gQ="
+  bucket: software-buildingrobotics
+  local-dir: wheel/dist
+  upload-dir: python_saml
```

Comment:
Can you change this to python3_saml? Just to keep it separate from the existing python-saml code we have in there